docs/user-guide/install.md

```diff
@@ -91,3 +91,30 @@ vsphere
 ID                  NAME                DESCRIPTION         ENABLED
 ```
 
+## Upgrade Instructions
+
+The volume driver can be upgraded or reinstalled any time with no impact to the user data. User data in volumes existing before an upgrade of the volume driver is retained as it is and is accessible post an upgrade.
+
+Our volume managed plugin is taking care of the control path (management layer) and doesn't deal with data path. So during the upgrade process, only management related operations will be affected and data will be intact. Management operations like create/attach/detach/delete volume using driver may run into an error for the time being until the upgrade process is finished.
+If the plugin is removed while volume is still attached to a vm, then volume will remain attached.
+
+We do not require user to stop containers to do plugin upgrade.
+
+Similarly, no change in behavior of vmgroup operations is expected due to volume driver upgrade.
+
+An upgrade shouldn't have to always require un-install and install of both driver or plugin. At times only one may change.
```

Comment:
L105 is confusing to me .. Please rephrase it. BTW, it should be our recommendation to user staying on the same release version. Can you please explain why do we need such information here?

Reply:
Please mention if user must always upgrade both service and plugin or only one. If both then is there an order in which that should happen? For example can containers be running when ESX service is upgraded? Because those may exit and leave the volumes attached to the VM. Forcing the user to bounce the plugin so it can detect and detach unused volumes.